src/components/TeamFounder/FounderProfile.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import * as S from "./FounderProfile.styles";
+// import { TeamMember } from "./TeamInformation";
+import { FounderInfo } from "./FounderInformation";
+
+import FounderProfileDescription from "./FounderProfileDescription";
+import SocialMedia from "components/SocialMedia/SocialMedia";
+import { SocialMediaColor } from "../../utility/SharedStyles";
+import { faLinkedinIn } from "@fortawesome/free-brands-svg-icons";
+
+type ProfileProps = {
+  key: number;
+  member: FounderInfo;
+  mobileView: boolean;
+  profilePic: string;
+  alt: string;
+};
+
+const Profile = (props: ProfileProps) => {
```

Comment:
Duplicate of src/components/TeamSection/Profile.tsx

Reply:
deleted FounderProfile, made changes to Profile.tsx to ensure it works for Founder